rust/Cargo.toml

```diff
@@ -20,7 +20,7 @@ libsecp256k1 = { version = "0.7.0", optional = true }
 hex = { version = "0.4.2", optional = true }
 crossbeam = { version = "0.8.1", optional = true }
 rapier3d = { version = "0.16.1", optional = true }
-bs58 = { version = "0.4.0", optional = true }
+bs58 = { version = "0.4.0", optional = true } 
```

Comment:
Looks like a parasite change: not sure why there should be a whitespace at the end.